src/BreweryScrapper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2022B;
+
+class BreweryScrapper
+{
+    public const SHOW = "show";
+    public const LIST = "list";
+    public const CLEAR = "clear";
+    public const EXIT = "exit";
+
+    public function getShowLabel($name): void
+    {
+        $lines = file(__DIR__ . "/../tests/stubs/table.txt");
+        $found = false;
+        foreach ($lines as $line) {
+            if (strpos($line, $name) !== false) {
+                $found = true;
+                echo $line;
+            }
+        }
+        if (!$found) {
+            echo "No match found";
+        }
+    }
+
+    public function getList(): void
+    {
+        $lines = file(__DIR__ . "/../tests/stubs/table.txt");
+        foreach ($lines as $line) {
+            $parts = explode("|", $line);
+            if (isset($parts[0]) && $parts[5] !== "Provider ") {
+                echo $parts[5];
+            } else {
+                echo " ";
+            }
+        }
+    }
+
+    public function getClear(): void
+    {
+        opcache_reset();
```

Comment:
OpCache should not be here. As a cache you can use some file, stored in the application directory. If for example, name of brewery exists in this file then you should use data from this file otherwise you need to find them via scrapper.

Please research some information about OpCache.
https://www.php.net/manual/en/book.opcache.php
https://support.cloud.engineyard.com/hc/en-us/articles/205411888-PHP-Performance-I-Everything-You-Need-to-Know-About-OpCode-Caches